src/test/java/it/pagopa/dss/SignatureServiceTest.java

```diff
@@ -0,0 +1,36 @@
+package it.pagopa.dss;
```

Comment:
do you think this coverage is enough?

Reply:
Still lack `addSignatureToPadesFile` coverage!

Reply:
Added new test and coverage report!

Reply:
brilliant!